### ISSUE TITLE
Suggestions for refactoring

### ISSUE THREAD

**Issue by SudharshanaSL** [shreyasrkarthik/AutomaticSlidesGeneration#3]
I think we need to rename the files and variable names. Follow a set of coding conventions. Feel free to comment. 
I will be re-factoring the text file names.

**Comment by shreyasrkarthik** [comment]
I agree some of our functions are in the abc_xyz format, and others are in abcXyz.
Same applies for files.
